### DISCUSSION TITLE
When I was adding bacnet device, it appeared that the device was connected successfully, but the label used could not be displayed. Any good suggestions? The simulators used are Bacnet.Room.Simulator and Yabe

### DISCUSSION THREAD

**Discussion by KOTP17** [frangoteam/FUXA#821] [Q&A]
When I was adding bacnet device, it appeared that the device was connected successfully, but the label used could not be displayed. Any good suggestions? The simulators used are Bacnet.Room.Simulator and Yabe

**Comment by unocelli** [comment]
Hi, we have to investigate. we tested with the ProtoSense simulator. is the Bacnet.Room.Simulator simulator freeware?

**Reply by mummyu** [reply]
yabe  Bacnet.Room.Simulator
